imagersite/imagersite/templates/registration/activation_email.txt

```diff
@@ -0,0 +1,5 @@
+Hey! You registered for our site. Here is your activation info.
+
+Activation Key: {{ activation_key }}
+Expiration Date: {{ expiration_date }}
+Site: {{ site }}
```

Comment:
this email should contain a link that takes the user to the site.  You can build that using the available information.  Take a look at what the `site` thing you are printing there actually contains.